Documentation/network/lb-ipam.rst

```diff
@@ -0,0 +1,382 @@
+.. only:: not (epub or latex or html)
```

Comment:
Comment: This is awesome documentation. Thanks so much for writing it up!

I know these things are a pain to write. I'm seriously impressed.